gaeguli/pipeline.c

```diff
@@ -586,15 +577,31 @@ _get_source_description (GaeguliPipeline * self)
   return g_string_free (result, FALSE);
 }
 
+static gchar *
+_get_stream_props_description (GaeguliPipeline * self)
+{
+  GString *stream_props = g_string_new (PIPEWIRE_NODE_STREAM_PROPERTIES_STR);
+
+  g_string_append_printf (stream_props, ",%s=%p_%d",
+      GAEGULI_PIPELINE_TAG, self, getpid ());
```

Comment:
Nitpick: I'd put PID first, instance pointer second in `gaeguli.pipeline_id` so that the values could be easily sorted by process :)